constructorio-client/src/test/java/io/constructor/client/ConstructorIOBrowseTest.java

```diff
@@ -92,6 +127,37 @@ public void BrowseShouldReturnAResultWithBrandFacets() throws Exception {
         assertTrue("browse result id exists", response.getResultId() != null);
     }
 
+    @Test
+    public void SearchShouldReturnAResultWithVariations() throws Exception {
```

Comment:
Should this be `BrowseShouldReturnAResultWithVariations`?

Reply:
yup, thanks for catching that. updated